test/change-stream.test.js

```diff
@@ -0,0 +1,87 @@
+describe('PersistedModel.createChangeStream()', function() {
+  describe('configured to source changes locally', function() {
+    before(function() {
+      var test = this;
+      var app = loopback({localRegistry: true});
+      var ds = app.dataSource('ds', {connector: 'memory'});
+      this.Score = app.model('Score', {
+        dataSource: 'ds',
+        changeDataSource: false // use only local observers
```

Comment:
Can we use a more specific name instead of `changeDataSource`? See https://github.com/strongloop/loopback/pull/1445#discussion_r33587585